pkg/csi_driver/node.go

```diff
@@ -79,16 +85,36 @@ func (s *nodeServer) NodePublishVolume(ctx context.Context, req *csi.NodePublish
 	// Mount source
 	source := fmt.Sprintf("%s:/%s", attr[attrIp], attr[attrVolume])
 
+	// FileSystem type
```

Comment:
Do we expect to support additional filesystem types in the future? (or, say, do we want to use this csi driver for all filesystem types supported by filestore?)

The pattern of hardcoding fstype according to the OS type seems a bit strange to me. Is this associated with any input and can be validated?

Reply:
Before I made any changes the driver only supported "nfs" (it was hardcoded). [mount_windows.go](https://github.com/kubernetes/kubernetes/blob/master/pkg/util/mount/mount_windows.go#L86) only supports CIFS.

I don't think it's associated with any input. Users don't get to choose the FS type in the case of FileStore.